### DISCUSSION TITLE
`call coc#config('hover', { 'previewMaxHeight': 50 })` do nothing

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#3943] [Q&A] [answered]
Still not enough space for the error message:

![image](https://user-images.githubusercontent.com/3514015/180307665-e36149e3-b48e-4e0b-9516-55ec2f79c8ec.png)

And it looks like a defect - such a great Vim plugin should have enough space to show messages out of the box.

**Comment by chemzqm** [comment, marked as answer]
You shoudl use `"diagnostic.floatConfig"` instead.

**Reply by vitaly-zdanevich** [reply]
Have `call coc#config('diagnostic', { 'floatConfig': {'maxHeight': 20, 'maxWidth': 120 }})` but still last line is cut.

**Reply by vitaly-zdanevich** [reply]
Solved by `call coc#config('diagnostic', { 'floatConfig': {'pumpheight': 20 }})`

**Reply by vitaly-zdanevich** [reply]
`border` doing nothing? I have this:

    call coc#config('diagnostic', { 'floatConfig': {'pumpheight': 20, 'maxWidth': 300, 'border': 'true' }})

but no border.
